RetroBar/Controls/TaskButton.xaml

```diff
@@ -84,6 +84,18 @@
                     </MenuItem.Icon>
                 </MenuItem>
                 <Separator />
+                <MenuItem Header="{DynamicResource end_task}"
+                          Click="EndTaskMenuItem_OnClick"
+                          Name="EndTaskMenuItem"
+                          Visibility="{Binding Source={x:Static utilities:Settings.Instance}, Path=ShowEndTaskButton, UpdateSourceTrigger=PropertyChanged, Converter={StaticResource boolToVisibilityConverter}}">
+                    <MenuItem.Icon>
+                        <TextBlock FontFamily="Marlett"
+                                   Text="&#x72;"
+                                   HorizontalAlignment="Center"
+                                   VerticalAlignment="Center"
+                                   FontWeight="Bold" />
+                    </MenuItem.Icon>
```

Comment:
Let's remove the icon for this item so that someone doesn't accidentally choose the wrong item based on icon.